### DISCUSSION TITLE
TerrariaServer.exe 1.4.3.6

### DISCUSSION THREAD

**Discussion by LANWrench** [Pryaxis/TShock#2584] [Q&A]
I'd like to get this working with the recent patch 1.4.3.6.

I tried replacing the TerrariaServer.exe in the TShock zip with the new one but that doesn't seem to work.

What are my options for getting this working?

**Comment by drunderscore** [comment]
~~This is an _experimental_ branch of TShock that utilizes OTAPI 3 and .NET 6. You cannot simply replace the single file -- all files must be replaced.~~

~~Additionally, it will _not_ work with existing plugins.~~

Update: Please read my updated reply below.

**Reply by LANWrench** [reply]
Awesome thank you.

**Reply by ShubhamKaudewar** [reply]
![image](https://user-images.githubusercontent.com/38852978/156897381-822b5c06-407b-41e3-aea2-cfa9f829d2ae.png)
Patch file of TShock 1.4.3.6 is worked for me. I am playing latest version of Terreria on AWS EC2

**Comment by drunderscore** [comment]
So sorry, I had accidentally gave out incorrect information. The experimental release was _not_ on OTAPI3.

It is still true that you need to replace _all_ the files with the updated ones, not just `TerrariaServer.exe`, which may have caused it to not work.
